libraries/Network/src/NetworkClient.h

```diff
@@ -0,0 +1,115 @@
+/*
+  Client.h - Base class that provides Client
```

Comment:
Maybe we should update this line to reflect the actual description of this file. I think as long as the copyright and the license stays untouched it should be fine. (might be wrong though)

Reply:
yeah, I am also a bit scared of touching other's headers, so I would rather not